hack/tools/go.mod

```diff
@@ -3,12 +3,31 @@ module sigs.k8s.io/kind/hack/tools
 go 1.12
 
 require (
-	github.com/client9/misspell v0.3.4
+	github.com/golangci/golangci-lint v1.17.2-0.20190714145355-d2b1eea2c617
 	github.com/spf13/pflag v1.0.3 // indirect
-	golang.org/x/lint v0.0.0-20190409202823-959b441ac422
+	golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529 // indirect
 	golang.org/x/tools v0.0.0-20190606174628-0139d5756a7d // indirect
-	honnef.co/go/tools v0.0.0-20190605142022-0a11fc526260
 	k8s.io/code-generator v0.0.0-20190311093542-50b561225d70
 	k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a // indirect
 	k8s.io/klog v0.3.0 // indirect
 )
+
+// deal with golangci-lint being broken
```

Comment:
https://github.com/golangci/golangci-lint/issues/595#issuecomment-524166777

Reply:
basically: go 1.13 is stricter about the timestamps matching the source when using a pseudo-version, the module proxy is equally strict now even in 1.12 / 1.11. golangci-lint needs to update their go.mod upstream, there are PRs open for this.